engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/docker-tags.controller.js

```diff
@@ -44,5 +44,17 @@ angular.module('Bastion.docker-tags').controller('DockerTagsController',
             }
             return versions.join(", ");
         };
+
+        $scope.getRepositoryNames = function (tag) {
+            var names = [];
+            var i;
+            for (i = 0; i < tag.repositories.length; ++i) {
```

Comment:
Just throwing this out there as the stylistic choice is up to you - but there's a mix of 2 styles in this method.

I think it could be rewritten like:
```
return tag.repositories.map(function(repo) {
  return repo.name;
}).filter(function(item, index) {
  return names.indexOf(item) >= index;
}).sort().join(', ');
```

Reply:
this works fine for me!